Add case study: CW20

rainsworth opened the pull request:
This pull request adds Collaborations Workshop 2020 - Online as a case study.

It includes sections on the following EPS topics:
- [x] Infrastructure
   - Platforms (Eventbrite, Zoom, Slack, Google Docs)
- [x] Timeline
   - 3 weeks to adapt (closed registration, contacted speakers/workshop facilitators, reorganised agenda, rewrote duties roster, got remaining staff/fellows registered, issued refunds, reopened registration, recruited extra helpers for delivery team, held pre-event meetings for delivery team and speakers, built the infrastructure, produced the slides)
- [x] Budget
- [x] Committees
- [x] Agenda
   - Session types (icebreakers, keynotes, lightning talks, discussion groups, collaborative ideas, mini-workshops)
   - Need to add Hackday
- [x] Publicity
- [x] Sponsorship
- [x] Venue
   - Conversations with venue about rescheduling
- [x] Catering
- [x] Event Roles
- [x] Risk Management
   - Zoom bombers
   - Zoom fatigue
   - COVID-19 considerations
   - Accommodation costs lost
   - CoC adaptation
- [x] Outputs and Outcomes Plan
   - Appropriate privacy policy and guidance
- [x] Closing Down the EPS
   - Feedback
   - Published reflections on event (blog posts)
   - Published speed blogs
   - Processed and published session recordings
   - Shared infrastructure documentation and resources
   - Guidance for online training
- [ ] tl;dr Lessons Learned section?

Feel free to add feedback at any time (are there any topics/sections that need adding?). I will update this description as I add sections and make commits.

Resolves: Issue #37

shoaibsufi commented:
Looking good!

shoaibsufi commented on docs/eog/case-studies-cw20-agenda.rst:
```diff
@@ -0,0 +1,76 @@
+We contacted speakers and workshop facilitators to ask if they were still able to deliver in the online setting. 
+In the end, two of the workshops dropped out and we were therefore able to shift the others around so that we had 16 workshops across four sessions (with four workshops in parallel for each session). 
+
+Because we were now unable to carry out the planned social programme (activities such visiting the Ulster Museum, a walk around the Belfast Botanic Gardens and of course the conference dinner), we were able to shorten the days and add in further breaks and changeover/buffer time. 
```
```suggestion
Because we were now unable to carry out the planned social programme (activities such visiting the Ulster Museum, a walk around the Belfast Botanic gardens and of course the conference dinner), we were able to shorten the days and add in further breaks and changeover/buffer time. 
```

shoaibsufi commented on docs/eog/case-studies-cw20.rst:
```diff
@@ -0,0 +1,42 @@
+
+The Collaborations Workshop (CW) is the annual flagship event of the Software Sustainability Institute. It brings together researchers, developers, innovators, managers, funders, publishers, leaders, and educators to explore and share best practice.
+
+On 5 March - just over three weeks before Collaborations Workshop 2020 (CW20) was due to start in Belfast, Northern Ireland - everything was in place. The programme had been finalised, keynote speakers announced, catering booked and accommodation assigned. There had been some issues - a group from Northern Italy had cancelled because they had been quarantined due to an outbreak of a novel coronav ...
```
```suggestion
On 5 March 2020 - just over three weeks before Collaborations Workshop 2020 (CW20) was due to start in Belfast, Northern Ireland - everything was in place. The programme had been finalised, keynote speakers announced, catering booked and accommodation assigned. There had been some issues - a group from Northern Italy had cancelled because they had been quarantined due to an outbreak of a novel coronavirus. At this point the number of reported cases in the UK was small, but things were moving rapidly. The clear message was that we couldn’t run the event as planned. 
```

shoaibsufi replied:
Aside: I just noticed the agenda - https://software.ac.uk/cw20/agenda should have a link to the speed blogs produced - perhaps using the tags - e.g. https://software.ac.uk/tags/cw20-speed-blog-posts

shoaibsufi commented on docs/eog/case-studies-cw20-infrastructure.rst:
```diff
@@ -0,0 +1,67 @@
+====================
+
+We had already organised registration for the in-person event through `Eventbrite <https://www.eventbrite.com/>`_. 
+We sent emails to registrants through Eventbrite to notify them of our decision making every step of the way. 
```
```suggestion
We emailed registrants through Eventbrite to notify them of the decisions that we were making and thereby keep the informed. 
```

shoaibsufi commented on docs/eog/case-studies-cw20-infrastructure.rst:
```diff
@@ -0,0 +1,67 @@
+
+We had already organised registration for the in-person event through `Eventbrite <https://www.eventbrite.com/>`_. 
+We sent emails to registrants through Eventbrite to notify them of our decision making every step of the way. 
+Because of the unconference nature of the event, we even asked for participant input to our contingency investigation. 
```
```suggestion
Because of the unconference nature of the event, we even asked for participant input to our contingency investigation; surveying them to see if they preferred the event to stay as an in person event, move to online or be cancelled. 
```

shoaibsufi commented on docs/eog/case-studies-cw20-infrastructure.rst:
```diff
@@ -0,0 +1,67 @@
+Because of the unconference nature of the event, we even asked for participant input to our contingency investigation. 
+As we worked to reformat the event and move it online, we gave registrants time to cancel and collect full refunds. 
+We also reduced the cost of the event (from £250 to £50) and issued partial refunds through Eventbrite to participants who wanted to stay and participate in the virtual event. 
+We took the partial refund route in order to maintain continuity with our registrants, but another option could have been to outright cancel the Eventbrite event, issue full refunds and start from scratch. 
```
Might be worth saying - why the other option might have been worth considering - I think there was a different fee impact if I remember.

shoaibsufi commented on docs/eog/case-studies-cw20-infrastructure.rst:
```diff
@@ -0,0 +1,67 @@
+Video conferencing
+^^^^^^^^^^^^^^^^^^^
+
+`Zoom <https://zoom.us/>`_ was very quickly identified as the video conferencing platform that we would use because we had the most experience with it and it had the functionality we needed (namely breakout rooms and easy recording of the event). 
```
```suggestion
`Zoom <https://zoom.us/>`_ was very quickly identified as the video conferencing platform that we would use because we had the most experience with it and it had the functionality we needed (scaling and stability with many video based participants, breakout rooms and easy recording of the event). 
```

shoaibsufi commented on docs/eog/case-studies-cw20-infrastructure.rst:
```diff
@@ -0,0 +1,67 @@
+The final resource that we needed to decide on was what platform to use for collaborative note-taking - a method for participant interaction and engagement that we learned from communities such as `Mozilla Open Leaders <https://foundation.mozilla.org/en/initiatives/mozilla-open-leaders/>`_, `rOpenSci <https://ropensci.org/commcalls/>`_ and `NL-RSE Meetups <https://nl-rse.org/pages/meetups.html>`_ ...
+Google Docs was the obvious choice for CW20, as we generate a lot of Google Docs through uCONFLY for the various sessions, and it also has the lowest barrier to accessibility (no need to log in or register and you do not need to know specific syntax or markdown language such as with `HackMD <https://hackmd.io>`_). 
+We created a note-taking document for each day which contained the full agenda and all relevant event information, a space for roll call, instructions and links (links to Zoom rooms for parallel sessions, Discussion group and Collaborative Ideas documents, etc.) as can be seen in this `example collaborative document <https://doi.org/10.6084/m9.figshare.12498257>`_. 
+There were spaces created within the document for people to take notes during the talks and to ask any questions; all of which could be preserved and referred back to in the future. 
```
```suggestion

There were spaces created within the document for people to take notes during the talks and to ask any questions; all of which could be preserved and referred back to in the future. 
```

shoaibsufi replied:
It's a long paragraph and I think it can be made into two at this point.

shoaibsufi commented on docs/eog/case-studies-cw20-infrastructure.rst:
```diff
@@ -0,0 +1,67 @@
+For CW20, we used uCONFLY purely as an admin tool by creating the templates for the Discussion, Collaborative Ideas and Mini-workshop sessions, and generated all of the needed instances ahead of time ourselves. 
+We then collated them via Google Sheets (see this `example spreadsheet <https://doi.org/10.6084/m9.figshare.12498278>`_) which we linked to in the day’s collaborative note-taking document/agenda. 
+
+Resources
```
It might be worth making the list into two sublist - one for hosts-only and one for everyone (i.e. hosts and participants) - I think it will make it clearer for those who are following along to use the case study.

shoaibsufi commented on docs/eog/case-studies-cw20-infrastructure.rst:
```diff
@@ -0,0 +1,67 @@
+Slack also allowed for a more human and informal engagement through the creation of channels such as ``#pets-at-cw20``, ``#kids-at-cw20`` and ``#outside-my-window-cw20`` for sharing photos of our pets, children and working from home environment during lockdown.
+
+
+Collaborative documenting
```
Is it worth mentioning the limit we reached with Google Docs - i.e. 100 connections before you had to make it public to support that many connections - and we could not do that as it would open the event to disruption. And perhaps a caveat that at the time of writing HackMD although requiring the knowledge of some syntax can support over 240+ connections.

shoaibsufi commented on docs/eog/case-studies.rst:
```diff
@@ -0,0 +1,12 @@
+   :maxdepth: 1
+   :caption: Sections:
+
+   case-studies-cw20
```
I think it is worth adding authorship to each case study - that person (or persons) can go on the main author list also - but I think given the somewhat standalone nature of the case study - I think that would be ok - what do you think @rainsworth ?

rainsworth replied:
Yes, that makes sense. I can't find in the restructured text primer how to e.g. add an author to this TOC where you've made the comment, so I've added authorship to the first page of the cw20 case study.

shoaibsufi replied:
That's fine

shoaibsufi commented on docs/eog/case-studies-cw20-timeline.rst:
```diff
@@ -0,0 +1,145 @@
+As days passed and the implications around the global pandemic became more clear, stakeholders were very understanding and accommodating. 
+Below is a breakdown of what we did. 
+
+Event Project Stage
```
The Event Project Stage is a container - it covers the stage when you get the big 'OK' to move forward with your event - so Pre-Event, During the Event and Post-Event are all contained in the Event Project Stage - so it's better to  put sub-headings in to reflect the sub-stages within the Event Project Stage.

rainsworth replied:
I have just removed those other headings, as heading levels in restructuredtext are not assigned to certain characters but the structure is determined from the succession of headings.

shoaibsufi commented on docs/eog/case-studies-cw20-timeline.rst:
```diff
@@ -0,0 +1,145 @@
+Agenda
+  - We reorganised agenda for online format by
+   
+    - Removing in-person social programme (walk in Belfast Botanical Gardens, visit to Ulster Museum, Drinks Reception, Conference Dinner).
```
This is the first time you mention that the event was originally meant to take place in Belfast (and perhaps it was worth mentioning that this would be the first CW taking place in Northern Ireland) - in the intro of the case study.

shoaibsufi commented on docs/eog/case-studies-cw20-timeline.rst:
```diff
@@ -0,0 +1,145 @@
+  - We set up a uCONFLY instance for management of Discussion Group Reports, Mini-workship and Demo session notes, Collaborative Ideas and Hack Day ideas which also involved
+   
+    - Creating Google Doc templates for each of these document types.
+    - Generating a number of each document type for dissemination during CW20 so that participants would not be responsible for yet another platform.
```
```suggestion
    - Generating a number of each document type for dissemination during CW20 so that participants would not have to login and use yet another platform.
```

shoaibsufi commented on docs/eog/case-studies-cw20-timeline.rst:
```diff
@@ -0,0 +1,145 @@
+  
+Event run up actions
+  - Closed registration immediately upon decision to move event online.
+  - Issued full refunds to registrants who were no longer able to attend up to 7 days before the event start date.
```
```suggestion
  - Issued full refunds to registrants who were no longer able to attend and informed us at least 7 days before the event start date.
```

shoaibsufi commented on docs/eog/case-studies-cw20-timeline.rst:
```diff
@@ -0,0 +1,145 @@
+Event Running Stage
+--------------------
+
+Month 0
```
Might be worth saying it ran smoothly according to plan and any funnies that happened - I don't know if making everyone co-host to allow mingling for the hackday team formation is worth mentioning and that is an area we hope to improve in the future - i.e. mingling in online workshops.

shoaibsufi commented on docs/eog/case-studies-cw20-timeline.rst:
```diff
@@ -0,0 +1,145 @@
+  - Announced availability of `Infrastructure resources <https://software.ac.uk/news/collaborations-workshop-2020-resources-now-available>`_.
+  
+Resources
+  - 10/12 speed blogs published.
```
Is is the case that all 12 will be published - or were there 12 sessions - and 10 made it as published blogs of the right quality (at the moment it reads as more maybe coming).

shoaibsufi commented on docs/eog/case-studies-cw20-budget.rst:
```diff
@@ -0,0 +1,29 @@
+We were extremely lucky that there were no additional costs for moving the event online - the University of Manchester (UoM) had already provided staff with Education licensed Zoom accounts which allowed for unlimited meeting duration for up to 300 people. 
+There were enough UoM staff members on the CW20 delivery team that we had enough Zoom Host accounts for the main plenary room plus 4 additional rooms in parallel for the Mini-workshop and demo sessions. 
+
+We did reduce the cost of registration from £250 to £50 as there would be no catering involved in an online event. 
```
```suggestion
We did reduce the cost of registration from £250 to £50 as there would be no catering or physical venue involved in an online event. 
```

shoaibsufi commented on docs/eog/case-studies-cw20-budget.rst:
```diff
@@ -0,0 +1,29 @@
+There were enough UoM staff members on the CW20 delivery team that we had enough Zoom Host accounts for the main plenary room plus 4 additional rooms in parallel for the Mini-workshop and demo sessions. 
+
+We did reduce the cost of registration from £250 to £50 as there would be no catering involved in an online event. 
+The reasons we didn't make the event free included the staff time that had gone into organising the event, to prevent a large number of no-shows on the day and because there were prizes involved in the Hack Day.
```
```suggestion
The reasons we didn't make the event free included the staff time that had gone into organising the event, to prevent a large number of no-shows on the day and because there were prizes involved.
```

shoaibsufi replied:
I took Hack Day out as it also applied to Collaborative Ideas and you mentioned it further down the page.

shoaibsufi commented on docs/eog/case-studies-cw20-agenda.rst:
```diff
@@ -0,0 +1,76 @@
+For in-person CWs, the days are long to maximise the opportunities for networking, collaboration, discussions, and solving problems across the research software landscape. 
+The agenda that we had spent months organising was packed full of talks, group sessions, social activities and mini-workshops - we originally had 18 mini-workshops scheduled to take place across five sessions.
+
+If we had organised CW20 as an online event from the beginning, there are many things we would have done differently - but since we found ourselves in the position of having to transition from an in-person event to an online one, we still wanted to incorporate as much of the original programme as possible. 
```
```suggestion
If we had organised CW20 as an online event from the beginning, there are many things we would have done differently - but since we found ourselves in the position of having to transition from an in-person event to an online one at short notice, we still wanted to incorporate as much of the original programme as possible. 
```

shoaibsufi commented on docs/eog/case-studies-cw20-agenda.rst:
```diff
@@ -0,0 +1,76 @@
+Keynote Talks (Informing)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Keynote speakers were contacted prior to the event to confirm the time of talk and format, and no time keeping was necessary on the day. 
```
```suggestion
Keynote speakers were contacted prior to the event to confirm the time of their talk and its format, and enable no additional time keeper being needed for their session on the day. 
```

shoaibsufi commented on docs/eog/case-studies-cw20-agenda.rst:
```diff
@@ -0,0 +1,76 @@
+Keynote speakers were contacted prior to the event to confirm the time of talk and format, and no time keeping was necessary on the day. 
+The Chair of the session introduced the speaker and reminded the participants that they could take notes and ask questions within the session’s section of the day’s collaborative notes where a link to the slides was also shared. 
+The speaker shared their presentation slides via `Screen Share in Zoom <https://support.zoom.us/hc/en-us/articles/201362153-Sharing-your-screen>`_, and the Host set `Spotlight Video <https://support.zoom.us/hc/en-us/articles/201362653-Spotlight-Video>`_ for the speaker so that they would be visible to all participants and in the recording. 
+During the Q&A, the Chair then relayed some of the questions from the notes to the speaker, and the remainder of the questions were answered in-line within the note-taking document.
```
```suggestion
During the Q&A, the Chair then relayed some of the questions from the notes to the speaker, and the remainder of the questions were answered in-line within the note-taking document after the session.
```

shoaibsufi commented on docs/eog/case-studies-cw20-agenda.rst:
```diff
@@ -0,0 +1,76 @@
+We pointed people to the different Zoom room links in the day’s collaborative notes and in the Slack channel, and someone stayed in the main room to help people get to where they wanted to go. 
+A lesson learned is to provide guidance to workshop facilitators that the workshops should include participant interaction for at least half of the time, and not just one long presentation. 
+
+You can find the full collection of `CW20 infrastructure and documentation resources on Figshare <https://doi.org/10.6084/m9.figshare.c.5026400>`_.
```
This sentence seems to float on it's own - it's not part of the Mini-workshops and Demos (Teaching) section - is it needed here? - if so perhaps give it it's own heading e.g. 'Infrastructure to help run the session' or put it at the end of the introductory paragraph on the page

shoaibsufi commented on docs/eog/case-studies-cw20-catering.rst:
```diff
@@ -0,0 +1,9 @@
+=====================
+
+After a discussion on 9 March 2020 with the local organisers at Queen's University Belfast, we agreed to postpone CW in Belfast to a future date, with no charges for the change to the room booking dates and catering. 
+Ultimately, Government restrictions and lockdowns would have prevented the event taking place in person anyways. 
```
```suggestion
Ultimately, further Government restrictions and lockdown measures throughout out March would have prevented the event taking place in person if we had decided to go ahead with an in person event. 
```

shoaibsufi commented on docs/eog/case-studies-cw20-event-roles.rst:
```diff
@@ -0,0 +1,32 @@
+CW20 Event Roles
+================
+
+It was clear that additional roles and duties were needed for an online event relative to an in-person event. The roles we needed included: 
```
It read like the roles that follow are just needed for an online event - or is at least a bit ambiguous to me - whereas it's clear that e.g. a Chair and a Twitter person are needed for in person events also. - I think the Four sets ... and Zoom participants managers - were the unique ones for the online.

shoaibsufi commented on docs/eog/case-studies-cw20-event-roles.rst:
```diff
@@ -0,0 +1,32 @@
+We also scheduled Zoom “Office Hours” for workshop facilitators to join and test the functionality of Zoom in preparation for their sessions and to ask any questions. 
+Finally, we had a Code of Conduct Committee call to walk through the incident response procedure, discuss potential violations that may occur during an online event and any additional risk mitigation we could put in place to prevent them. 
+
+You can find the full collection of `CW20 infrastructure and documentation resources on Figshare <https://doi.org/10.6084/m9.figshare.c.5026400>`_.
```
Again - it seems a bit strange to put this here - I can understand if someone came in via a google search let's say - perhaps add a sub-heading - e.g. resources. I can see why you put this here - but it's a bit jarring as the last sentence - but then again perhaps we can have a chat about what we want on each page and it is a bit weird with guides done this way - each page has to almost standalone.

shoaibsufi commented on docs/eog/case-studies-cw20-outputs-and-outcomes-plan.rst:
```diff
@@ -0,0 +1,52 @@
+
+As part of our post-event feedback survey, we asked participants how confident their understanding was of each of the CW20 themes in the context of their research/work, both before and after the event. 
+In general, participants were more confident about their understanding of Open Research, Data Privacy and Software Sustainability after attending CW20.
+
```
For all of these before and after pairs it's worth saying what the average score was before and after - and then a %-age improvement - it just makes it easier to understand the shift in understanding - e.g. on average CW20 improved attendees confidence from 3.2/5 to 4.1/5 a huge 30% increase <-- note numbers are made up and not checked just here for illustration : )

shoaibsufi commented on docs/eog/case-studies-cw20-publicity.rst:
```diff
@@ -0,0 +1,52 @@
+
+- `Highlights from Collaborations Workshop 2020 <https://software.ac.uk/blog/2020-04-21-highlights-collaborations-workshop-2020>`_: a short reflection post 
+- `SSI Collaborations Workshop 2020: Remote unconference experience and notes <https://software.ac.uk/blog/2020-04-29-ssi-collaborations-workshop-2020-remote-unconference-experience-and-notes>`_: a participant's experience of CW20
+- `CW20: how to move an event online in three weeks <https://software.ac.uk/blog/2020-05-18-cw20-how-move-event-online-three-weeks>`_: a longer reflection post 
```
```suggestion
- `CW20: how to move an event online in three weeks <https://software.ac.uk/blog/2020-05-18-cw20-how-move-event-online-three-weeks>`_: a longer reflective piece 
```

shoaibsufi commented on docs/eog/case-studies-cw20-publicity.rst:
```diff
@@ -0,0 +1,52 @@
+
+We published several blog posts after the event, including:
+
+- `Highlights from Collaborations Workshop 2020 <https://software.ac.uk/blog/2020-04-21-highlights-collaborations-workshop-2020>`_: a short reflection post 
```
```suggestion
- `Highlights from Collaborations Workshop 2020 <https://software.ac.uk/blog/2020-04-21-highlights-collaborations-workshop-2020>`_: a short reflective piece 
```

shoaibsufi commented on docs/eog/case-studies-cw20-risk-management.rst:
```diff
@@ -0,0 +1,265 @@
+   - No change in what we pay for staff/fellows hotels, although some may now not attend
+
+2. Move CW20 online
+
+   - We would have to ask sponsors
+   - We might lose some registrations
+   - We would have to change what we charge from £250 for a full in-person ticket to £50 for an online pass to attend (to go towards invested staff time and prizes)
+   - There would be no additional costs as we already had the video conferencing infrastructure in place
```
Is it worth mentioning here - the potential loss of money from cancelling the venue, accommodation and reimbursing peoples travel that we had agreed?

shoaibsufi commented on docs/eog/case-studies-cw20-timeline.rst:
```diff
@@ -0,0 +1,147 @@
+^^^^^^^^^^^^^^^^^^^^
+
+- 11/12 discussion groups' `speed blogs published <https://software.ac.uk/tags/cw20-speed-blog-posts>`_ - the last one may be published in the future if the group members decide to.
+
```
Do we want a Month 6 (September 2020)

+ EOG case study completed and reviewed

shoaibsufi commented on docs/eog/case-studies-cw20.rst:
```diff
@@ -0,0 +1,44 @@
+
+Case study authors: Rachael Ainsworth (`ORCID <http://orcid.org/0000-0003-2591-9462>`__) and Shoaib Sufi (`ORCID <https://orcid.org/0000-0001-6390-2616>`__)
+
+The Collaborations Workshop (CW) is the annual flagship event of the Software Sustainability Institute. It brings together researchers, developers, innovators, managers, funders, publishers, leaders, and educators to explore and share best practice.
```
We might send people a link to the case study - it might be the first thing they see on EOG - so I think linking Software Sustainability Institute to www.software.ac.uk would be a good thing to do - even though it's a repeat of the link from the intro of the EOG

shoaibsufi commented on docs/eog/case-studies-cw20.rst:
```diff
@@ -0,0 +1,44 @@
+
+CW is a highly interactive event with around 80-100 participants, and is designed to facilitate meeting as many new people as possible. With a global pandemic forcing us to stay at home, could we successfully recreate the experience online, and with only three weeks to prepare? Given the amount of work that had already been put into organising the event, the decision was made on 10 March to move  ...
+
+Here is how we did it.
```
```suggestion
Here is how we did it. Note, the case study takes place just before our decision to go online and covers the steps in the Event Project Stage of the main EOG. We do not go into details about the face to face CW20 event that was planned. The case study runs until the resources after the workshop are were made available. 
```

rainsworth commented:
Hi @shoaibsufi - I've addressed all your comments 🎉 

Please let me know if there is anything else you would like added or expanded on - otherwise this is ready for final review/merge/comms team.

shoaibsufi reviewed (approved):
Looks good!
